src/eoapi/vector/eoapi/vector/sql/mercator.sql

```diff
@@ -0,0 +1,131 @@
+CREATE OR REPLACE FUNCTION find_items(
+    IN geom geometry,
+    IN queryhash text,
+    IN items_limit int DEFAULT 10000,
+    IN _scanlimit int DEFAULT 10000,
+    IN _timelimit interval DEFAULT '5 seconds'::interval,
+    OUT id text
+) RETURNS setof text AS $$
+DECLARE
+    search searches%ROWTYPE;
+    curs refcursor;
+    _where text;
+    query text;
+    iter_record items%ROWTYPE;
+    exit_flag boolean := FALSE;
+    counter int := 1;
+    scancounter int := 1;
+    remaining_limit int := _scanlimit;
+BEGIN
+    SELECT * INTO search FROM searches WHERE hash=queryhash;
+
+    IF NOT FOUND THEN
+        RAISE EXCEPTION 'Search with Query Hash % Not Found', queryhash;
+    END IF;
+
+    _where := format('%s AND ST_Intersects(geometry, %L::geometry)', search._where, ST_Transform(geom, 4326));
+
+    FOR query IN SELECT * FROM partition_queries(_where, search.orderby) LOOP
```

Comment:
right now we are returning the items which intersects the geometry but we only need the `count`